### ISSUE TITLE
False positive for semicolon rule

### ISSUE THREAD

**Issue by chrismbarr** [palantir/tslint#2206]
### Bug Report

- __TSLint version__: `4.4.2`
- __TypeScript version__: `2.1.6`
- __Running TSLint via__: `gulp-tslint 7.1.0`

#### TypeScript code being linted

```ts
class myClass {

    public foo = () => { }

    public bar = () => {
      
    }

}
```

with `tslint.json` configuration:

```json
"semicolon": [ true, "always", "ignore-interfaces" ],
```

#### Actual behavior

The method `foo` on a single line reports an invalid "Missing semicolon" violation, but the method `bar` on multiple lines does not report a violation (correctly).

#### Expected behavior

The number of lines a method takes up should not affect whether a semicolon is required or not.  **I would expect a single-line method to not require a semicolon** just like the rest.

**Comment by nchen63** [comment]
duplicate of https://github.com/palantir/tslint/issues/1949

**Comment by chrismbarr** [comment]
@nchen63 Perhaps I am misunderstanding, but this does not appear to be a duplicate of that issue.  

Improper semicolons **are** being reported on class method members **unless they are declared on a single line* , in which case it reports that a semicolon is **required** which is incorrect.

Here's a better example:

    class myClass {

        public foo1 = () => { }  //<-- reports "missing semicolon" (incorrect)

        public bar1 = () => {
      
        }   //<-- nothing reported  (correct)

        public foo2 = () => { };  //<-- nothing reported (incorrect, this should not be required)

        public bar2 = () => {
      
        };   //<-- reports "misplaced semicolon" (correct)

    }

**Comment by nchen63** [comment]
Previously, all of those cases required semicolons. https://github.com/palantir/tslint/issues/1476 was a change that allowed missing semicolons for the multiline case, so this is by design
